build.sbt

```diff
@@ -1,21 +1,18 @@
 import wartremover.Wart
 import sbt.project
 
-val scala3 = "3.0.1"
+val scala3 = "3.1.1"
```

Comment:
Scala 3 versions are finally going to be compatible? :D

Reply:
That's right! Scala 3 (and Scala 2.13) use an intermediate format called Tasty, https://docs.scala-lang.org/scala3/guides/tasty-overview.html, instead of .class files, and Scala 3.1 can read Tasty files generated by Scala 3.0, and even Tasty files generated by Scala 2.13.

It's only forwards compatible though, which lead to some issues for library authors, but that's getting fixed with https://contributors.scala-lang.org/t/improving-scala-3-forward-compatibility/5298 so that you can choose to generate 3.0 Tasty files even with the Scala 3.1 compiler, and so on.